app/src/main/java/com/hedvig/app/feature/keygear/KeyGearCoverageBottomSheet.kt

```diff
@@ -0,0 +1,116 @@
+package com.hedvig.app.feature.keygear
+
+import android.app.Dialog
+import android.os.Bundle
+import android.os.Parcelable
+import com.hedvig.app.R
+import com.hedvig.app.ui.fragment.RoundedBottomSheetDialogFragment
+import com.hedvig.app.util.apollo.ThemedIconUrls
+import com.hedvig.app.util.svg.buildRequestBuilder
+import kotlinx.android.parcel.Parcelize
+import kotlinx.android.synthetic.main.dialog_key_gear_coverage.*
+
+class KeyGearCoverageBottomSheet : RoundedBottomSheetDialogFragment() {
+
+    private val requestBuilder by lazy { buildRequestBuilder() }
+
+    override fun getTheme() = R.style.NoTitleBottomSheetDialogTheme
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val dialog = super.onCreateDialog(savedInstanceState)
+        dialog.setContentView(R.layout.dialog_key_gear_coverage)
+
+        requestBuilder.load(mock.icon.iconByTheme(requireContext()))
+            .into(dialog.icon)
+
+        dialog.title.text = mock.title
+        dialog.description.text = mock.description
+
+        dialog.bulletPointsRecyclerView.adapter =
+            CoverageInfoAdapter(mock.boxes, requestBuilder)
+
+        return dialog
+    }
+
+    companion object {
+        const val TAG = "coverageBottomSheet"
+
+        fun newInstance(): KeyGearCoverageBottomSheet {
```

Comment:
Let's make this take the data as its `arguments`, so that we can quickly plug in the real data once we have it 🙂